content/blog/release-1-8-0.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Rivet 1.8.0 release
+description: Rivet 1.8.0 is now available. This release adds border removal utility classes and fixes bugs related to the button, dropdown, and collapsible components.
+date: "2021-02-17"
+tags:
+    - "minor release"
+excludeFromIndex: true
+author:
+  email: "scanmurr@iu.edu"
+  name: "Scott Anthony Murray"
+---
+Rivet version `1.8.0` is now available! This release adds border removal utility classes and fixes bugs related to the button, dropdown, and collapsible components.
+
+## Border removal utility classes
+This release adds utility classes that allow you to remove borders from any or all sides of an element. For more information about these new classes, see the [border utilities documentation]({{< ref "../components/utilities/border.md" >}}).
+
+## Button focus outline colors
+This release fixes an issue where a button component's outline color when focused did not match the overall button color if the danger or success variants were used.
+
+## Dropdown error handling
+This releases fixes an issue that caused Rivet to throw an error if a dropdown component was opened or closed programmatically after it had been removed from the DOM.
+
+## Collapsible icon width
```

Comment:
Technically the collapsible is a separate package/repo, so we'll need to make sure we do that release before we publish this post as well. At that point we can link to the collapsible release on Github. For example: https://github.com/indiana-university/rivet-collapsible/releases/tag/v0.2.5